### ISSUE TITLE
Error when creating osz

### ISSUE THREAD

**Issue by IIRamix** [Corne2Plum3/fnf2osumania#32]
I get this error when exporting:
Traceback (most recent call last):
  File "fnf_converter.py", line 672, in exportAsOsz
  File "fnf_converter.py", line 200, in exportOsuFile
  File "fnf_converter.py", line 345, in getNotesFromPlayer
ValueError: invalid literal for int() with base 10: 'Show Song'
Any solutions?

**Comment by Corne2Plum3** [comment]
Which song? Can you put the song and the chart file here?

**Comment by IIRamix** [comment]
[demise.json](https://github.com/Corne2Plum3/fnf2osumania/files/14126613/demise.json) 
the chart from mario madness v2

**Comment by Corne2Plum3** [comment]
Never seen something like this -_-

```JSON
					[
						16105.2315789474,
						1,
						236.842105263158
					],
					[
						15157.8947368421,
						-1,
						"Show Song",
						"",
						""
					]
```

**Comment by Corne2Plum3** [comment]
Fixed in version 1.3.1. Download it, it should work
